### PR TITLE
Translate Quiz Extras and add test

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#26.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

https://user-images.githubusercontent.com/80920858/122105310-3e80e300-cdde-11eb-9638-c2e11bccb3fd.mp4

### CODE REVIEW

File: _sources/quiz/QuizExtras.rst [hunk 1]
```diff
@@ -7,142 +7,172 @@ Quiz - Extras
...
 
-   Problema de Josephus. Imagine que temos n pessoas dispostas em círculo. Suponha que as pessoas estão numeradas 1 a n no sentido horário. |br|
-   Começando com a pessoa de número 1, percorra o círculo no sentido horário e elimine cada m-ésima pessoa en quanto o círculo tiver duas ou mais pessoas. |br|
-   Qual o número do sobrevivente? |br|
+.. activecode:: qExtra_1
+    :nocodelens:
 
-   ~~~~
-   def josephus(n,m):
+    **Problema de Josephus**. Imagine que temos ``n`` personas dispuestas en un círculo. Suponga que esas personas están numeradas de 1 a ``n`` en sentido horario.
```
Comment (LeoCumpli21) [inline _sources/quiz/QuizExtras.rst:16]:
```suggestion
    **Problema de Josephus**. Imagine que tenemos ``n`` personas dispuestas en un círculo. Suponga que esas personas están numeradas de 1 a ``n`` en sentido horario. 
```

Comment (y-martinez) [inline _sources/quiz/QuizExtras.rst:16]:
```suggestion
    **Problema de Josephus**. Imagine que tenemos ``n`` personas dispuestas en un círculo. Suponga que esas personas están numeradas de 1 a ``n`` en sentido horario. 
```

File: _sources/quiz/QuizExtras.rst [hunk 2]
```diff
@@ -7,142 +7,172 @@ Quiz - Extras
...
-   Qual o número do sobrevivente? |br|
+.. activecode:: qExtra_1
+    :nocodelens:
 
-   ~~~~
-   def josephus(n,m):
+    **Problema de Josephus**. Imagine que temos ``n`` personas dispuestas en un círculo. Suponga que esas personas están numeradas de 1 a ``n`` en sentido horario. 
+    Comenzando con la persona número **1**, ejecute el círculo en sentido horario y elimine cada ``m``-ésima persona, siempre y cuando el círculo 
+    tenga dos o más personas. El reto es desarrollar la función ``josephus`` que toma a ``n`` y a ``m`` como parámetros, ambos números enteros positivos.
+    La función debe devolver el número en el círculo del sobrevivente. |br| |br|
```
Comment (y-martinez) [inline _sources/quiz/QuizExtras.rst:19]:
```suggestion
    La función debe devolver el número de la persona en el círculo que sobrevivió. |br| |br|
```

File: _sources/quiz/QuizExtras.rst [hunk 3]
```diff
@@ -7,142 +7,172 @@ Quiz - Extras
...
 
-   ~~~~
-   def tiles(n):
+Ejercicio 5
+-----------
+
+.. activecode:: qExtra_5
+    :nocodelens:
+
+    Se dan ``n`` baldosas con dimensión de 10cm x 10cm. Con ellas, usted debe ensamblar un conjunto de cuadrados usando todas las baldosas dadas. Para ello, debe seguir algunas reglas:
```
Comment (y-martinez) [inline _sources/quiz/QuizExtras.rst:149]:
```suggestion
    Se dan ``n`` baldosas con dimensión de 10cm x 10cm. Con ellas, usted debe ensamblar un conjunto de cuadrados usando todas las baldosas dadas. Para ello, debe seguir las siguientes reglas:
```

**Review by y-martinez — APPROVED** [review]
I left some changes but LGTM :)

File: _sources/quiz/QuizExtras.rst [hunk 4]
```diff
@@ -7,142 +7,172 @@ Quiz - Extras
...
+Ejercicio 5
+-----------
+
+.. activecode:: qExtra_5
+    :nocodelens:
+
+    Se dan ``n`` baldosas con dimensión de 10cm x 10cm. Con ellas, usted debe ensamblar un conjunto de cuadrados usando todas las baldosas dadas. Para ello, debe seguir las siguientes reglas:
+
+    - El primer cuadrado que se construya debe usar la mayor cantidad de baldosas posibles.
+    - Si quedan bladosas, el siguiente cuadrado también debe construirse con la mayor cantidad de las baldosas que quedaron.
```
Comment (angelasofiaremolinagutierrez) [inline _sources/quiz/QuizExtras.rst:152]:
```suggestion
    - Si quedan baldosas, el siguiente cuadrado también debe construirse con la mayor cantidad de las baldosas que quedaron.
```

File: _sources/quiz/QuizExtras.rst [hunk 5]
```diff
@@ -7,142 +7,172 @@ Quiz - Extras
...
 
+Ejercicio 4
+-----------
 
-   ~~~~
-   def grainsum(n):
+.. activecode:: qExtra_4
+    :nocodelens:
 
+    En el libro “El Hombre que Calculaba”, de *Malba Tahan*, un personaje quería ganar los granos de trigo que estaban distribuídos sobre un tablero de ajedrez del siguiente modo:
```
Comment (angelasofiaremolinagutierrez) [inline _sources/quiz/QuizExtras.rst:121]:
```suggestion
    En el libro “El Hombre que Calculaba”, de *Malba Tahan*, un personaje quería ganar los granos de trigo que estaban distribuidos sobre un tablero de ajedrez del siguiente modo: 
```

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
Some grammar mistakes but it is ready to merge! :)

**Review by NicolasSandoval — APPROVED** [review]
LGTM!

**Pull request merged**